xlators/mgmt/glusterd/src/glusterd-svc-mgmt.c

```diff
@@ -517,3 +517,43 @@ glusterd_muxsvc_conn_init(glusterd_conn_t *conn, glusterd_svc_proc_t *mux_proc,
     }
     return ret;
 }
+
+/*
+ * A generic function replacing two functions,
+ * glusterd_bitdsvc_start and glusterd_scrubsvc_start
+ * wherein both do the same set of operations.
+ */
```

Comment:
The function name you gave to `glusterd_genericsvc_start` already conveys this info.
LGTM anyways.

Reply:
Ack. Just added a note so that we know what this function replaces. I didn't replace the function in quota. Probably will take it as a separate patch and see if there are any more duplicates.